prime-router/src/main/kotlin/azure/SubmissionFunction.kt

```diff
@@ -83,6 +88,8 @@ class SubmissionFunction(
 
     /**
      * An Azure Function that is triggered at the `/api/submissions/` endpoint
+     * This endpoint is meant for use by a User.  It assumes the user belongs to a single Organization.
+     * DO NOT USE - DEPRECATED - DELETE THIS
```

Comment:
@whytheplatypus and I discussed removing this endpoint completely.   I left it here for now.